DEBUG 2021-09-14 08:37:21.968 UTC: 	 CRAB Client version: v3.210825
DEBUG 2021-09-14 08:37:21.969 UTC: 	 Running on: Linux lxplus773.cern.ch 3.10.0-1160.36.2.el7.x86_64 #1 SMP Wed Jul 21 11:57:15 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2021-09-14 08:37:21.969 UTC: 	 Executing command: 'kill'
DEBUG 2021-09-14 08:37:21.969 UTC: 	 Found CRAB cache file /afs/cern.ch/user/y/yian/.crab3
ERROR 2021-09-14 08:37:21.970 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/cern.ch/work/y/yian/work/GZ/CMSSW_10_6_20/src/PhysicsTools/NanoAODTools/WWG/2017/crab/crab2017/crab_WWg_signal_full
ERROR 2021-09-14 08:37:21.970 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210825.00/bin/crab", line 151, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210825.00/bin/crab", line 136, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210825.00/lib/CRABClient/Commands/SubCommand.py", line 270, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210825.00/lib/CRABClient/Commands/SubCommand.py", line 650, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.210825.00/lib/CRABClient/Commands/SubCommand.py", line 512, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/cern.ch/work/y/yian/work/GZ/CMSSW_10_6_20/src/PhysicsTools/NanoAODTools/WWG/2017/crab/crab2017/crab_WWg_signal_full
INFO 2021-09-14 08:37:21.975 UTC: 	 Log file is /afs/cern.ch/work/y/yian/work/GZ/CMSSW_10_6_20/src/PhysicsTools/NanoAODTools/WWG/2017/crab/crab2017/crab.log
